templates/500.html

```diff
@@ -1,4 +1,4 @@
-{% extends "templates/one-column.html" %}
+{% extends "templates/one-column_no_nav.html" %}
```

Comment:
Why did you remove the navigation on the 500 page? This is get the user stuck there. Just because the current page is 500'ing it doesn't mean other pages wont work as expected.

Reply:
@anthonydillon oddly, it wasn't executing, and showing empty navigation - and it didn't parse the {{asset_server}} variable - see https://github.com/canonical-websites/www.ubuntu.com/issues/2591

Reply:
Makes sense, dynamic (YAML) navigation will not work in a server error situation. If we hard code it, it will get out of date. So the Ubuntu logo is ok for now.